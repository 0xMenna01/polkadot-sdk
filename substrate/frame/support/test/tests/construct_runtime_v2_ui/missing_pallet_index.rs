#[frame_support::construct_runtime_v2]
mod runtime {
    #[frame::runtime]
    pub struct Runtime;

    #[frame::pallets]
    #[frame::derive(RuntimeCall)]
    pub struct Pallets {
        System: frame_system
    }
}

fn main() {}
