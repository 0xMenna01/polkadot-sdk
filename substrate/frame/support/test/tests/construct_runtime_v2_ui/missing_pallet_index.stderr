error: Invalid pallet declaration, expected a #[frame::pallet_index]
 --> tests/construct_runtime_v2_ui/missing_pallet_index.rs:9:9
  |
9 |         System: frame_system
  |         ^^^^^^
